### PR TITLE
paper release details added

### PR DESCRIPTION
**Pull request by finalelement**

### CODE REVIEW

**Review by holgerroth — APPROVED** [review]
Looks good!

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ The repository provides a collection of vision language models, benchmarks, and
 
 ## 💡 News
 
+- [2024/12/04] The arXiv version of VILA-M3 is now available [here](https://arxiv.org/pdf/2411.12915).
```
Comment (holgerroth) [inline README.md:10]: Please link to the arxiv page and not the pdf directly https://arxiv.org/abs/2411.12915

**Review by holgerroth — CHANGES_REQUESTED** [review]
Sorry. One link should be updated.

File: README.md (hunk 1 above)
Comment (finalelement) [reply README.md]: Fixed.

**Review by holgerroth — APPROVED** [review]
Thanks!

**Pull request merged**